code/game/gamemodes/cult/cult_items.dm

```diff
@@ -91,4 +118,149 @@
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "cultstone"
+	used = 0
+	increment = 5
+	max = 40
+	prefix = "darkened"
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/unholywater
+	name = "flask of unholy water"
+	desc = "Toxic to nonbelievers; this water renews and reinvigorates the faithful of a cult."
+	icon_state = "holyflask"
+	color = "#333333"
+	list_reagents = list("unholywater" = 40)
+
+/obj/item/clothing/glasses/night/cultblind
+	desc = "May the master guide you through the darkness and shield you from the light."
+	name = "zealot's blindfold"
+	icon_state = "blindfold"
+	item_state = "blindfold"
+	darkness_view = 8
+	flash_protect = 1
+
+/obj/item/clothing/glasses/night/cultblind/equipped(mob/user, slot)
+	..()
+	if(!iscultist(user))
+		to_chat(user, "<span class='cultlarge'>\"You want to be blind, do you?\"</span>")
+		user.unEquip(src, 1)
+		user.Dizzy(30)
+		user.Weaken(5)
+		user.eye_blind = 30
+
+/obj/item/device/shuttle_curse
+	name = "cursed orb"
+	desc = "You peer within this smokey orb and glimpse terrible fates befalling the escape shuttle."
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state ="bluespace"
+	color = "#ff0000"
+	var/global/curselimit = 0
+
+/obj/item/device/shuttle_curse/attack_self(mob/user)
+	if(!iscultist(user))
+		user.unEquip(src, 1)
+		user.Weaken(5)
+		to_chat(user, "<span class='warning'>A powerful force shoves you away from [src]!</span>")
+		return
+	if(curselimit > 1)
+		to_chat(user, "<span class='notice'>We have exhausted our ability to curse the shuttle.</span>")
+		return
+	if(shuttle_master.emergency.mode == SHUTTLE_CALL)
+		var/cursetime = 1500
+		var/timer = shuttle_master.emergency.timeLeft(1) + cursetime
+		shuttle_master.emergency.setTimer(timer)
+		to_chat(user,"<span class='danger'>You shatter the orb! A dark essence spirals into the air, then disappears.</span>")
+		playsound(user.loc, "sound/effects/Glassbr1.ogg", 50, 1)
+		qdel(src)
+		sleep(20)
+		var/global/list/curses
+		if(!curses)
+			curses = list("A fuel technician just slit his own throat and begged for death. The shuttle will be delayed by two minutes.",
+			"The shuttle's navigation programming was replaced by a file containing two words, IT COMES. The shuttle will be delayed by two minutes.",
+			"The shuttle's custodian tore out his guts and began painting strange shapes on the floor. The shuttle will be delayed by two minutes.",
+			"A shuttle engineer began screaming 'DEATH IS NOT THE END' and ripped out wires until an arc flash seared off her flesh. The shuttle will be delayed by two minutes.",
+			"A shuttle inspector started laughing madly over the radio and then threw herself into an engine turbine. The shuttle will be delayed by two minutes.",
+			"The shuttle dispatcher was found dead with bloody symbols carved into their flesh. The shuttle will be delayed by two minutes.",
+			"A bunch of lightbulbs exploded around Steve and he can't see the console. The shuttle will be delayed by two minutes")

```

Comment:
this should be `Steve repeatedly touched a lightbulb until his hands fell off. The shuttle will be delayed by two minutes.`